packages/docs/getting-started/using-a-ledger-wallet.md

```diff
@@ -0,0 +1,111 @@
+# Using a Ledger Wallet
+
+This section shows how to set up a [Ledger] (https://www.ledger.com/) Nano S or X hardware wallet and get Celo addresses. 
+
+{% hint style="danger" %}
+The steps below require technical knowledge. You should be comfortable with the Command Line Interface (CLI) and understand the basics of how cryptographic network accounts work.
+{% endhint %}
+
+## Install the Celo Application
+
+Start by installing the Celo application and setting a PIN on your Ledger device by following steps 1 and 2 on https://www.ledger.com/start/.
+
+{% hint style="danger" %}
+Make sure to back up both the PIN and the recovery phrase, if you lose them you lose access to your Celo assets with no recovery possible.
+{% endhint %}
+
+Open the Ledger Live App on your computer and follow the instructions on the screen.
+
+Once in the app, click the settings gear icon (top right).
+
+![](https://storage.googleapis.com/celo-website/docs/ledger-settings.png)
+
+Click on the **Experimental features** menu.
+
+Turn on **Developer mode.** 
+
+![](https://storage.googleapis.com/celo-website/docs/ledger-settings-dev-mode.png)
+
+Exit the Settings menu by clicking on **Manager** on the left hand side bar.
```

Comment:
@kobigurk correct me if I'm wrong, but we still need to set the provider to `4`, right?

Reply:
Nope! It's published.